### ISSUE TITLE
The plugin fails when p2 artifacts are used.

### ISSUE THREAD

**Issue by niport** [reficio/p2-maven-plugin#149]
If I try the provided p2 example with version 1.3.0 of the plugin I get this error:
```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:19 min
[INFO] Finished at: 2019-11-27T10:51:30+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.reficio:p2-maven-plugin:1.3.0site (default-cli) on project example-p2-site: Execution default-cli of goal org.reficio:p2-maven-plugin:1.2.0-SNAPSHOT:site failed: java.lang.RuntimeException: Cannot resolve [org.junit_4.11.0.v201303080030.jar] from any given repository -> [Help 1]
```

**Comment by niport** [comment]
I cloned the current master version (1.4.0-SNAPSHOT), tried to use it but the error I got was the same. 
After some debugging the issue was related to the fact that in file:

https://github.com/reficio/p2-maven-plugin/blob/master/src/main/groovy/org/reficio/p2/resolver/eclipse/impl/DefaultEclipseResolver.groovy

The field repository.type (line 66) was always "default" despite the actual value of the repository layout tag.

After some investigations I supposed that the bug was related to maven (I was using mvn version 3.6.1 on Windows 10). Updating to maven 3.6.3 fixed the issue.

In other words, the p2-maven-plugin p2 repository artifacts integration *does not work with maven 3.6.1*, at least on Windows 10.

I guess that worth to mention this in the README.md file.
